db/migrate/20240723030506_add_post_id_index_to_user_histories.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddPostIdIndexToUserHistories < ActiveRecord::Migration[7.1]
```

Comment:
Usually for indicies we need to add them concurrently, and also using `disable_ddl_transaction!`. Here is a good example:

https://github.com/discourse/discourse/blob/e020888b0a30832697aa4735eaaba35f4668b641/db/migrate/20240304030429_topic_id_on_incoming_email_index.rb